tests/test_password.py

```diff
@@ -26,7 +26,9 @@ def test_success_change_password(self, requests_mock, cli_runner):
             json={"detail": "New password has been saved."},
         )
 
-        result = cli_runner.invoke(change_command, ["--password", new_password])
+        result = cli_runner.invoke(
+            change_command, ["--password", new_password, "--password", new_password]
```

Comment:
Although codecov wasn't happy, this isn't a correct usage of the package, so please return it back to
`result = cli_runner.invoke(change_command, ["--password", new_password])`